src/ray/gcs/format/gcs.fbs

```diff
@@ -73,6 +73,10 @@ table TaskInfo {
   actor_counter: int;
   // True if this task is an actor checkpoint task and false otherwise.
   is_actor_checkpoint_method: bool;
+  // If this is an actor task, then this will be populated with all of the new
+  // actor handles that were forked from this handle since the last task on
+  // this handle was submitted.
+  new_actor_handles: [string];
```

Comment:
The Java CI failure is because when we change `TaskInfo` definition, the corresponding Java file needs to be changed as well. 

Unfortunately, for now, generating this file isn't automatic. Because the generated file lacks a certain API and needs some manual patch. I proposed adding the required API to flatbuffers. But they're not willing to do that (https://github.com/google/flatbuffers/issues/5092).

For this PR, you can apply this patch to fix https://gist.github.com/raulchen/1fcecac74ddb809b836ab9bbf9f9e178.

We'll add a script to simplify this process.

Reply:
The TLDR explanation of this issue is: we define some ObjectID fields as `string` in `fbs`. But these fields are actually `byte array`s. For C++ and Python 2, `string` and `byte array` are equivalent. But for Java and Python 3, they're different. 

Flatbuffers Java API will decode the bytes with UTF8 if a field is defined as `string`, but the content is actually not UTF8-decodable. Python API doesn't do that for now, but may do in the future. Because Flatbuffers prefers to distinguish strings and byte arrays. If so, our Python code will face the same problem. 

An alternative solution is to store hex strings in fbs. But that might require substantial code changes.

Reply:
Ah, thanks!